tools/ci/src/main.rs

```diff
@@ -314,6 +320,24 @@ fn main() {
         );
     }
 
+    if checks.contains(Check::TEST_CHECK) {
+        let mut args = vec!["--workspace", "--examples"];
```

Comment:
This checks examples but not tests

```suggestion
        let mut args = vec!["--workspace", "--tests"];
```

Reply:
Thank you, good catch!